youtube_dl/extractor/nrk.py

```diff
@@ -479,6 +513,15 @@ class NRKTVSeriesIE(NRKTVSerieBaseIE):
     _VALID_URL = r'https?://(?:tv|radio)\.nrk(?:super)?\.no/serie/(?P<id>[^/]+)'
     _ITEM_RE = r'(?:data-season=["\']|id=["\']season-)(?P<id>\d+)'
     _TESTS = [{
+        'url': 'https://tv.nrk.no/serie/blank',
+        'info_dict': {
+            'id': 'blank',
+            'title': 'Blank',
+            'description': 'Da Markus dro fra bygda var han den kuleste fyren med den fineste dama. Etter kort tid er han tilbake. Og ingenting er som før. Norsk dramaserie.',
```

Comment:
^

Reply:
Is there something wrong with this line, or is it to show what to do on the line above?

Reply:
Same as in previous review.

Reply:
Nothing changed.

Reply:
I clearly don't understand whats missing here.

Reply:
Hide it behind md5 as you did with the first test. What's not clear?

Reply:
That really didn't click in my head until you pointed it out, god i feel stupid